README.md

```diff
@@ -37,6 +37,12 @@ To run Roller, use:
 
 ## Testing
 
+First, make sure you have the latest version of all the dependencies:
```

Comment:
but what if the main branch code has different dependencies than the latest rollapp version?

Reply:
Good point. Should I add a `verify-versions` command to make sure your installed dependencies binaries match the code, and show which one doesn't match if any?

Reply:
Maybe just add a command/instructions for downloading and building the dependencies from the compile_locally.sh?